bin/launcher.py

```diff
@@ -0,0 +1,12 @@
+"""
+Main script to use the model with lightning CLI
+Training with fit and infer with predict
+"""
+
+from py4cast.cli import cli_main
+
+# python bin/launcher.py fit --config config/CLI/trainer.yaml --config config/CLI/poesy.yaml --config config/CLI/halfunet.yaml
+# python -m pdb bin/launcher.py predict --ckpt_path /scratch/shared/py4cast/logs/test_cli/last.ckpt --config config/CLI/trainer.yaml --config config/CLI/poesy.yaml --config config/CLI/halfunet.yaml
```

Comment:
you could put put these commands in the above docstring, in a "exemple usage" section pour instance